### DISCUSSION TITLE
Amazon rest API to create product for my seller central

### DISCUSSION THREAD

**Discussion by chprasad92** [amzn/selling-partner-api-models#3146] [General]
Hello, 

I've been trying to find amazon rest API from which we can create Product and update product.

If anyone can help me with any documentation that would be real help. 

Regards, 
Chandan

**Comment by NetRushJacob** [comment]
This would be the [Listings Items API](https://github.com/amzn/selling-partner-api-docs/tree/main/references/listings-items-api)
Or use the Feeds API with various product feeds (XML or flat file).
